### DISCUSSION TITLE
How to stop rebuilding after changing build type?

### DISCUSSION THREAD

**Discussion by ashkanaev** [microsoft/vscode-cpptools#7910] [Q&A]
Hi, there ! 

I got separated building directories for release and debug build type. 
` 
 "cmake.buildDirectory" : "${workspaceRoot}/build.${buildKit}.${buildType}",
`
But every time after changing build type debug -> release or release -> debug, i supposed to building whole project again. 
Is't possible to disable force rebuilding?

**Comment by sean-mcmanus** [comment]
I don't know...maybe ask https://github.com/microsoft/vscode-cmake-tools/issues ?
